MQ2FeedMe.cpp

```diff
@@ -126,214 +312,370 @@ void ListTypes(const std::list<std::string>& list)
 void Execute(PCHAR zFormat, ...)
 {
 	char zOutput[MAX_STRING]={0}; va_list vaList; va_start(vaList,zFormat);
-	vsprintf_s(zOutput,zFormat,vaList); if(!zOutput[0]) return;
-	DoCommand(GetCharInfo()->pSpawn,zOutput);
+	vsprintf_s(zOutput, zFormat, vaList);
```

Comment:
You cleaned this line up but not the line above. But I would question whether you need an Execute function at all (and based on your change below I believe you removed the only usage of it already).

Reply:
This whole function can be deleted and calls can be replaced with `DoCommandf`